.github/workflows/ci.yaml

```diff
@@ -47,10 +56,77 @@ jobs:
           echo "Chart version is $version"
           echo "CHART_VERSION=$version" >> $GITHUB_ENV
           echo "CHART_VERSION=$version" >> $GITHUB_OUTPUT
-        working-directory: charts/vaas
+        working-directory: vaas-helm/charts/vaas
 
       - name: Run chart-testing (lint)
-        run: ct lint --validate-maintainers=false --charts charts/vaas
+        run: ct lint --validate-maintainers=false --charts vaas-helm/charts/vaas
+
+      - name: Install Minikube
+        uses: manusa/actions-setup-minikube@v2.13.0
+        with:
+          minikube version: v1.34.0
+          kubernetes version: v1.30.0
+          driver: docker
+          start args: --cpus 4 --memory 8192
+
+      - name: Set up kubectl
+        uses: azure/setup-kubectl@v3
+        with:
+          version: 'latest'
+
+      - name: Install Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: v3.13.3
+
+      - name: Enable ingress
+        run: |
+          minikube addons enable ingress
+          minikube kubectl -- wait --namespace ingress-nginx \
+            --for=condition=ready pod \
+            --selector=app.kubernetes.io/component=controller \
+            --timeout=120s
+
+      - name: Deploy Helm Chart
+        run: |
+          cat <<EOF > values-local.yaml
+          global:
+            secret:
+              dockerconfigjson: "${{ secrets.DOCKER_CONFIG_JSON }}"
+
+          cloud:
+            hashLookup:
+              enabled: false
+            allowlistLookup:
+              enabled: false
+          EOF
+          helm dep up vaas-helm/charts/vaas
+          helm upgrade --install vaas vaas-helm/charts/vaas -f values-local.yaml -n vaas --create-namespace --debug
+
+      - name: Verify Pods
+        run: |
+          sleep 10
+          minikube kubectl -- get pods --namespace vaas || true
+          minikube kubectl -- wait --for=condition=ready pods --namespace vaas --timeout=300s --all
+
+      - name: Run Java SDK Example
+        run: |
+          sudo echo "$(minikube ip) vaas" | sudo tee -a /etc/hosts
+          curl -s "https://get.sdkman.io" | bash
+          source "$HOME/.sdkman/bin/sdkman-init.sh"
+          sdk install java 21.0.3-tem
+          sdk use java 21.0.3-tem
```

Comment:
Not really a fan of grabbing Java like this, won't a simple apt-get install do the job as well?